kafka/consumer.go

```diff
@@ -122,6 +121,63 @@ func (c *Consumer) Unassign() (err error) {
 	return nil
 }
 
+// IncrementalAssign adds the specified partitions to the current set of
+// partitions to consume.
+// The new partitions must not be part of the current assignment.
+func (c *Consumer) IncrementalAssign(partitions []TopicPartition) (err error) {
```

Comment:
I assume you can specify offsets here? there's no comment about this and it doesn't follow from the typing.